causallearn/search/FCMBased/lingam/CAMUV.py

```diff
@@ -297,7 +297,7 @@ def get_K(X, width_x):
     return K, Kc
 
 
-def hsic_gam(X=None, Y=None, alph=None, width_x=None, width_y=None, K=None, Kc=None, L=None, Lc=None, mode=None,
+def hsic_gam(X=None, Y=None, alpha=None, width_x=None, width_y=None, K=None, Kc=None, L=None, Lc=None, mode=None,
```

Comment:
Thanks for pointing out these typos! I will comment on some unnecessary (potentially functional) changes like this one. Would you mind removing these before I merge the PR? Thanks!

[unnecessary]

(This does not really mean the change is 'unnecessary'. Just would like to take a safer choice :))

Reply:
done, force pushed